### DISCUSSION TITLE
How to disable font become bold when hovering the cursor over a tab?

### DISCUSSION THREAD

**Discussion by congcongkuo** [Jelmerro/Vieb#542] [Q&A] [answered]
Normal state:
<img width="467" alt="Screenshot 2024-01-06 at 17 53 59" src="https://github.com/Jelmerro/Vieb/assets/72879519/953e05ef-4997-46bd-bebd-f6d6d985ab53">

While hovering:
<img width="391" alt="Screenshot 2024-01-06 at 17 54 08" src="https://github.com/Jelmerro/Vieb/assets/72879519/bc14d03b-7154-4825-baa1-84facd39c961">

I want to disable this feature.

**Comment by Jelmerro** [comment, marked as answer]
Sure thing, you can disable this with custom themes. You can add a new colorscheme in one of the folders documented in `:help colorscheme`, but the default one is `~/.vieb/colors`, where you can add new CSS files that change the colorscheme. Since the default theme will remain active, it will take a few override rules to disable it, this will disable it once put in a new file in the colors folder:
```css
#tabs > span:hover {font-weight: normal !important;}
```
Let me know if you need more information or help with it :)
